### PR TITLE
Adding documentation for System.Net.WebSockets APIs released in 3.0

### PR DESCRIPTION
**Pull request by jozkee**
APIs documented:
* ClientWebSocket
  * ReceiveAsync(System.Memory{System.Byte}, ...)
  * SendAsync(System.ReadOnlyMemory{System.Byte}, ...)

I extracted the documentation from the previously existing overloads, just changed the `summary` a bit to try to disambiguate.
Added missing exceptions on existing APIs of SendAsync and ReceiveAsync as well.

### CODE REVIEW

**Comment by Jozkee** [comment]
@dotnet/ncl 
@rpetrusha @mairaw @carlossanlop

File: xml/System.Net.WebSockets/ClientWebSocket.xml [hunk 1]
```diff
@@ -473,11 +475,22 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="buffer">The region of memory to receive the response.</param>
+        <param name="cancellationToken">A cancellation token used to propagate notification that this  operation should be canceled.</param>
```
Comment (carlossanlop) [inline xml/System.Net.WebSockets/ClientWebSocket.xml]: Double space:

```suggestion
        <param name="cancellationToken">A cancellation token used to propagate notification that this operation should be canceled.</param>
```

**Comment by Jozkee** [comment]
Fixed double-spacing in other two summaries.
@mairaw LGMT.

**Comment by mairaw** [comment]
Good catch! I'll merge as soon as the build finishes.

**Pull request merged**